src/main/java/com/minecolonies/coremod/colony/buildings/BuildingMiner.java

```diff
@@ -284,7 +286,9 @@ public AbstractJob createJob(final CitizenData citizen)
     @Override
     public boolean neededForWorker(@Nullable final ItemStack stack)
     {
-        return Utils.isPickaxe(stack) || Utils.isShovel(stack) || Utils.isAxe(stack);
+        return ItemStackUtils.hasToolLevel(stack, ToolType.PICKAXE, TOOL_LEVEL_WOOD_OR_GOLD, getMaxToolLevel())
```

Comment:
I don't know, who use the pickaxe ?

- Builder
- Miner

Who doesn't ?

- LumperJack
- D-man
- Gards x2
- Baker